frontend/src/screens/MigrateNodeSuccess.tsx

```diff
@@ -18,8 +18,8 @@ export function BackupNodeSuccess() {
                   <Save className="w-6 h-6" />
                 </div>
                 <span>
-                  Your Alby Hub has been successfully backed up and saved to
-                  your filesystem.
+                  Your Alby Hub has been successfully migrated and saved to your
```

Comment:
This is very wrong. Both the description and the title.

title should be `Alby Hub Migration File Saved`
description: `Your Alby Hub migration file has been successfully saved to your filesystem.`